students/idcrickmore/lesson03/list_lab.py

```diff
@@ -1,37 +1,138 @@
-#!/usr/bin/ python3
+#!/usr/bin/env python3
+
+# Establish a global 'fruit' list variable
+fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+
 
 def series_1():
-    fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+
+    # This function modifies the global 'fruit' variable
+    print("Task 4")
+    global fruit
     print("the list is {}".format(fruit))
-    
-    fruit_response = input("What's another kind of fruit? > ")
-    
-    # change response type from string to list
-    fruit_response = [fruit_response]
-    
+
+    # ask user for a fruit and store it as a list
+    fruit_response1 = [input("What's another kind of fruit? > ")]
+
     # concatenate fruit list and response
-    fruit = fruit + fruit_response
+    fruit = fruit + fruit_response1
     print(fruit)
-    
+
     # count the list then ask the user for a number from 1 to x and save as int
     fruit_count = len(fruit)
     num_response = fruit_count + 1
     while (num_response > fruit_count) or (num_response < 1):
-        num_response = int(input("choose a number from 1 to {} > ".format(fruit_count)))
-        
+        num_response = int(input("choose a number from 1 to {} > "
+                           .format(fruit_count)))
+
     # concatenate the response using the 'fruit' list and user input
-    print("Number {} is {}".format(num_response,fruit[num_response-1]))
-    
-    fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+    print("Number {} is {}".format(num_response, fruit[num_response-1]))
+
     # add two more fruits
-    two_more_fruit = ["Persimmon", "Avocado"]
-    
+    fruit_response2 = [input
+                       ("What's a second kind of fruit you'd like to add? > ")]
+    fruit_response3 = [input
+                       ("What's a third kind of fruit you'd like to add? > ")]
```

Comment:
if you're not going to need fruit_response3 as a list then don't create list so that later it is simply `fruit.insert(0, fruit_response3)`

Reply:
These are great comments, thank you. 

Just so I understand the grading/workflow process, should I make these changes to the script and make another push to github?

IDC

Reply:
great question. grading wise, you will get a full grade for completing the assignment. 
when we leave suggestions/recommendations it is for your learning benefit, and yes you could do updates and commit into an open PR or create a new one (depending if one is still open or not).

For this particular PR - I would like to merge it as soon as possible as it is getting too big to manage. Ideally PRs should be small containing single feature.